### PR TITLE
enc28j60: Fixes and improvements

### PR DESCRIPTION
**Pull request by bthebaudeau**
See the commit messages for details.

### CODE REVIEW

File: dev/enc28j60/enc28j60.c [hunk 1]
```diff
@@ -128,17 +127,38 @@
...
 /*---------------------------------------------------------------------------*/
 static uint8_t
+is_mac_mii_reg(uint8_t reg)
+{
+  /* MAC or MII register (otherwise, ETH register)? */
+  switch(bank) {
+  case ERXTX_BANK:
+  case EPKTCNT_BANK:
+  default:
+    return 0;
```
Comment (tonnenpinguin) [inline dev/enc28j60/enc28j60.c]: Probably a personal preference, but I would put these three cases to the bottom of the switch statement as it improves readability if the default case is the last one.

**Comment by bthebaudeau** [comment]
Done.

**Comment by bthebaudeau** [comment]
I approve this PR. If nobody disapproves within one month, I will merge it. I'm just not certain whether this rule also applies if the PR author is the merging maintainer.

File: dev/enc28j60/enc28j60-ip64-driver.c [hunk 2]
```diff
@@ -85,12 +85,14 @@ PROCESS_THREAD(enc28j60_ip64_driver_process, ev, data)
   PROCESS_BEGIN();
 
   while(1) {
-    etimer_set(&e, 1);
-    PROCESS_WAIT_EVENT();
     len = enc28j60_read(ip64_packet_buffer, ip64_packet_buffer_maxlen);
     if(len > 0) {
+      etimer_set(&e, (len + 15) >> 4);
```
Comment (fros4943) [inline dev/enc28j60/enc28j60-ip64-driver.c]: This modification requires further explanation.

Comment (bthebaudeau) [inline dev/enc28j60/enc28j60-ip64-driver.c]: It is explained in the commit message:
30b3041e72b48b314442685adbf1a09a08b8bdfc

**Comment by simonduq** [comment]
@bthebaudeau ?

**Comment by bthebaudeau** [comment]
@simonduq @oliverschmidt @fros4943 So what do you want me to do for this PR following our discussion about the last commit? Should I just drop the last commit, then merge this PR and open an issue pointing to this thread?

**Pull request merged**